docs: dhus documentation (#840)
CESNET/perun-services: docs/concepts/send.md

# SEND script

> A bash script that transfers the generated data (as a .tar.gz archive) to the target machine.

Reviewer: bash/perl/python, ... whichever

it needn't pass .tar.gz at all - it can call external API for example

> A bash script that transfers the generated data to the target machine (e.g. by API call, tar, ssh, etc.).

Reviewer 2: ```suggestion
A script that transfers the generated data to the target machine (e.g. by API call, tar, ssh, etc.).
```
does not have to be bash

> The file name must match the service name.

Reviewer 2: see above

In a production environment, stored in `[path to engine]/send/`

> Each service script currently has to run a generic send script that takes care of everything. Eventually it will be

Reviewer: ```suggestion
Some services currently run a generic send script that takes care of everything. Eventually it will be
```

Reviewer: can add link to the generic_send file

> possible to remove the send scripts for each service altogether. The current form of all send scripts is as follows:

Reviewer: ```suggestion
possible to remove the send scripts for such services altogether. The current form of the simple send scripts is as follows:
```
